shared/src/commonMain/kotlin/org/hyperskill/app/step/presentation/StepReducer.kt

```diff
@@ -1,33 +1,71 @@
 package org.hyperskill.app.step.presentation
 
+import org.hyperskill.app.step.domain.analytic.StepViewedHyperskillAnalyticEvent
+import org.hyperskill.app.step.domain.model.StepRoute
 import org.hyperskill.app.step.presentation.StepFeature.Action
 import org.hyperskill.app.step.presentation.StepFeature.Message
 import org.hyperskill.app.step.presentation.StepFeature.State
-import org.hyperskill.app.step.domain.analytic.StepViewedHyperskillAnalyticEvent
+import org.hyperskill.app.step_completion.presentation.StepCompletionFeature
+import org.hyperskill.app.step_completion.presentation.StepCompletionReducer
 import ru.nobird.app.presentation.redux.reducer.StateReducer
 
-class StepReducer : StateReducer<State, Message, Action> {
+class StepReducer(private val stepRoute: StepRoute, private val stepCompletionReducer: StepCompletionReducer) : StateReducer<State, Message, Action> {
```

Comment:
Too long line 
```kotlin
class StepReducer(
    private val stepRoute: StepRoute,
    private val stepCompletionReducer: StepCompletionReducer
) : StateReducer<State, Message, Action> {
```

Reply:
@ivan-magda can we add checking this via ktlint?

Reply:
Not sure, maybe after [ALTAPPS-490](https://vyahhi.myjetbrains.com/youtrack/issue/ALTAPPS-490)